source/_layouts/base.liquid

```diff
@@ -58,20 +58,24 @@
     </p>
     <![endif]-->
 
-    {% typogr %}{% render 'header', alerts: alerts %}{% endtypogr %}
+    {%- unless is_playground -%}
+      {% typogr %}{% render 'header', alerts: alerts %}{% endtypogr %}
+    {%- endunless -%}
```

Comment:
Is there a good reason for the playground header to be inside of `<main>` instead of up here? If not, I'd suggest something like:

```suggestion
    {% typogr -%}
      {%- if is_playground -%}
        {% render 'playground_header', title: title %}
      {%- else -%}
        {% render 'header', alerts: alerts %}
      {%- endif -%}
    {%- endtypogr %}
```

Reply:
My thought was that the playground header was most analogous to the title, so kept it close there. But this code is more straightforward, and I don't think there's a significant semantic reason for that position.